src/components/Skeleton/Skeleton.tsx

```diff
@@ -68,27 +67,31 @@ const SkeletonContainer = styled(Div)`
         }
     `
       : ''
-  }
+    }
   `}
 `;
 
 export type SkeletonProps = {
   StyledContainer?: StyledSubcomponentType;
   StyledShimmer?: StyledSubcomponentType;
 
-  containerProps?: SubcomponentPropsType;
+  SkeletonProps?: SubcomponentPropsType;
   shimmerProps?: SubcomponentPropsType;
 
   children?: React.ReactNode;
   color?: string;
   isLoading?: boolean;
+
+  skeletonRef?: React.RefObject<HTMLElement>;
```

Comment:
I know it's out of scope, but while you're here could you add a `containerRef`?  That would be the last thing missing on the API for this component.